src/test/java/org/zalando/nakadi/domain/SubscriptionLocksCleaner.java

```diff
@@ -0,0 +1,41 @@
+package org.zalando.nakadi.domain;
+
+import org.apache.zookeeper.ZKUtil;
+import org.apache.zookeeper.ZooKeeper;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class SubscriptionLocksCleaner {
```

Comment:
Please document additional VM arguments that should be used to run this thing.

Reply:
Done.

Reply:
https://github.com/zalando/nakadi/pull/1057/commits/e082c8999aeb1a9b06aa71b2586e2491147e54ee